### PR TITLE
Fixed an issue where multiple widgets could not be expanded on initial loading

### PR DESCRIPTION
**Pull request by tk1024**
英語が苦手なので日本語で失礼します。

初回ロード時に2つ以上のウィジェットを展開しようとすると`react-instagram-embed-script`idのエレメントが存在していて且つ`window.instgrm`が存在しない状況があり展開に失敗する問題があましたので修正致しました。
恐らく #21 の問題の解決になるかと思います。

Because I am not good at English, I am rude in Japanese.

When trying to expand more than two widgets at the time of initial loading, there was a problem that the elements of `react-instagram-embed-script`id existed and there was no `window.instgrm` and there was a problem that the deployment failed I fixed it.
I think that it will probably be a solution to the problem of #21.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -13,26 +13,25 @@ export type Props = {
   onAfterRender: () => void,
   onFailure: () => void,
   protocol: string,
-  injectScript: boolean
```
Comment (sugarshin) [inline src/index.js]: Why did you remove? It is become breaking changes.

Comment (tk1024) [reply src/index.js]: Thank you for review.
I'll rewrite to remove breaking changes.

**Comment by tk1024** [comment]
I made minimal changes, so I think that breaking changes has ceased.
Plaese, check it.

**Comment by sugarshin** [comment]
@tk1024 Sorry late reply... I approved. Thank you for your contribution! I will merge to master also publish new version to npm.
